src/megatron/bridge/training/utils/train_utils.py

```diff
@@ -477,62 +489,90 @@ def training_log(
                 writer.add_scalar(metric, value, iteration)
             if wandb_writer:
                 wandb_writer.log(runtime_report, iteration)
+            if mlflow_logger and runtime_report:
+                mlflow_logger.log_metrics(_sanitize_mlflow_metrics(runtime_report), step=iteration)
         if logger_config.log_l2_norm_grad_to_tensorboard:
             l2_report = report_l2_norm_grad(model)
             for metric, value in l2_report.items():
                 writer.add_scalar(metric, value, iteration)
             if wandb_writer:
                 wandb_writer.log(l2_report, iteration)
+            if mlflow_logger and l2_report:
```

Comment:
```suggestion
            if mlflow_logger:
```

For consistency with wandb logic.